sympy/stats/stochastic_process_types.py

```diff
@@ -54,6 +55,27 @@ def _set_converter(itr):
         raise TypeError("%s is not an instance of list/tuple/set."%(itr))
     return itr
 
+def _sym_sympify(arg):
+    """
+    Converts an arbitrary expression to a type that can be used inside SymPy.
+    As generally strings are unwise to use in the expressions,
+    it returns the Symbol of argument if the string type argument is passed.
+
+    Parameter
+    =========
+    arg
+        The parameter to be converted to be used in Sympy.
```

Comment:
There should be a blank line under `====`. There should be a colon after args and the description should be on the same line:
https://docs.sympy.org/1.5.1/documentation-style-guide.html#parameters-section

Reply:
And also, it should be "Parameters" even if there is only one.